### DISCUSSION TITLE
Hi! I want to configure wezterm such that i dont have to split the panes and open tabs to adjust according to my needs everytime!

### DISCUSSION THREAD

**Discussion by theAnuragMishra** [wezterm/wezterm#6385] [Ideas]
I dont want tmux like sessions which keep processes running. I just want that there's some way to open wezterm into a layout i have configured (by passing some args maybe?)
something like wezterm -a layout1. i should be able to configure the layouts in probably .wezterm.lua itself

**Comment by andreasludvig** [comment]
Maybe have a look here: https://wezfurlong.org/wezterm/config/lua/gui-events/gui-startup.html
Sounds like that is what you're looking for.

**Reply by theAnuragMishra** [reply]
Tbh I don't understand this entire `wezterm start` thing

**Reply by andreasludvig** [reply]
Check `wezterm --help` in your terminal. Im no expert (rather a noob), but to my understanding all we need to know is, that when wezterm starts (ie opens the first GUI/terminal window), it sends an event called 'gui-startup'. You can use this event as shown in the doc links i provided, to have wezterm do something specific when this happens. Ie., when wezterm starts, I want wezterm to open with this layout.
